ras_party/controllers/account_controller.py

```diff
@@ -410,7 +410,10 @@ def resend_verification_email(party_uuid, session):
     if not respondent:
         raise RasError(NO_RESPONDENT_FOR_PARTY_ID, status=404)
 
-    _send_email_verification(party_uuid, respondent.email_address)
+    if respondent.pending_email_address != '':
```

Comment:
Probably better as `if not respondent.pending_email_address` So it deals with `Null` values too

Reply:
If there's no pending email then it comes through as an empty string rather than null so doing `if not respondent.pending_email_address` wouldn't work.

Reply:
`if not` will pick up on empty string, but yeah if its just an empty string can be left like this i guess

Reply:
Done 👍